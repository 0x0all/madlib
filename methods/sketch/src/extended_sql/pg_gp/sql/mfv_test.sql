create schema madlib;
\i sketches_drop.sql
\i sketches.sql
set search_path to "$user",public,madlib;

-- Basic methods
select frequent_values(i,5) 
  from (select * from generate_series(1,100) union all select * from generate_series(10,15)) as T(i);
select frequent_values(substring(name from '^.*/'), 9) from pg_timezone_names;
select frequent_values(NULL::bytea,5) from generate_series(1,100);