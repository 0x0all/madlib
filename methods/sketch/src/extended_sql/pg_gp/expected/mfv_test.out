set search_path to "$user",public,madlib;
-- Basic methods
select frequent_values(i,5) 
  from (select * from generate_series(1,100) union all select * from generate_series(10,15)) as T(i);
             frequent_values             
-----------------------------------------
 [10: 2] [11: 2] [12: 2] [13: 2] [14: 2]
(1 row)

select frequent_values(utc_offset,5) from pg_timezone_names;
                                 frequent_values                                  
----------------------------------------------------------------------------------
 [@ 0: 54] [@ 1 hour: 50] [@ 2 hours: 46] [@ 5 hours ago: 44] [@ 4 hours ago: 40]
(1 row)

select frequent_values(NULL::bytea,5) from generate_series(1,100);
 frequent_values 
-----------------
 
(1 row)

